### PR TITLE
prefix  refreshing when loading model

### PR DESCRIPTION
**Pull request by Shamanou**
Now the prefixes will load and render when a persisted model is loaded.

Commit also includes style updates

### CODE REVIEW

File: extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js [hunk 1]
```diff
@@ -26,143 +26,115 @@ function RdfSchemaAlignmentDialog(schema) {
...
-          });
-
+  
+  elmts._load_skeleton.click(function(e) {
+    e.preventDefault();
+    rdfSkeletonListDialog = new RdfSkeletonListDialog(function(rdfSkeleton) {
+      oldSchema = self._schema;
+      self._originalSchema = JSON.parse(rdfSkeleton['skeleton']);
+      self._schema = cloneDeep(self._originalSchema);
+      self._prefixesManager = new RdfPrefixesManager(self, self._schema.prefixes);
```
Comment (Shamanou) [inline extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js:85]: This is the main change it was needed to renew the _prefixesManager

File: extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js [hunk 2]
```diff
@@ -26,143 +26,115 @@ function RdfSchemaAlignmentDialog(schema) {
...
+                everythingChanged: true
+              });
+            }
+          }
+        });
+      });
+      self._originalSchema = theProject.overlayModels.rdfSchema;
+      self._schema = cloneDeep(self._originalSchema);
+      self._nodeUIs = [];
+      self._prefixesManager = new RdfPrefixesManager(self, self._schema.prefixes);
```
Comment (Shamanou) [inline extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js:128]: same here but for the append functionality

File: extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js [hunk 3]
```diff
@@ -26,143 +26,115 @@ function RdfSchemaAlignmentDialog(schema) {
...
...</ul>' + '<div id="rdf-schema-alignment-tabs-schema">' + '<div class="rdf-scheme-dialog-subheader"><table><tr><td><span style="display:block;width:150px;">Available Prefixes:</span></td><td><div class="rdf-schema-prefixes" bind="rdf_schema_prefixes"></div></td></tr></table></div>' + '<div class="schema-alignment-dialog-canvas rdf-schema-alignment-dialog-canvas"></div>' + '<div class="rdf-schema-alignment-body-footer"><a bind="add_another_root_node" href="#">Add another root node</a><a bind="_save_skeleton" href="#" style="float:right">&nbsp;&nbsp;&nbsp;&nbsp;Save&nbsp;&nbsp;&nbsp;&nbsp;</a>' + '<a bind="_share_skeleton" href="#" style="float:right">&nbsp;&nbsp;&nbsp;&nbsp;Share&nbsp;&nbsp;&nbsp;&nbsp;</a><a bind="_load_skeleton" href="#" style="float:right">&nbsp;&nbsp;&nbsp;&nbsp;Load&nbsp;&nbsp;&nbsp;&nbsp;</a>' + '</div>' + '</div>' + '<div id="rdf-schema-alignment-tabs-preview" style="display: none;">' + '<div class="rdf-scheme-dialog-subheader">This is a sample <code>Turtle</code> representation of (up-to) the <em>first 10</em> rows</div>' + '<div class="rdf-schema-alignment-dialog-preview" id="rdf-schema-alignment-dialog-preview"></div>' + '</div>' + '</div>').appendTo(body);
```
Comment (kburger) [inline extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js:68]: Ginormous oneliners do not improve readability/maintainability 👎

File: extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js (hunk 1 above)
Comment (kburger) [inline extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js:85]: Just wondering, does this work for an append as well?

**Review by kburger — APPROVED** [review]
Some minor comments, functionality wise it looks good. I am missing the agreement we made for splitting style changes from changes in logic. Do try to remember that.

**Pull request merged**